### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by cbjeukendrup**
Resolves: https://github.com/musescore/MuseScore/issues/25491

Scherm­afbeelding 2024-11-13 om 23 58 42

Try it out at https://github.com/cbjeukendrup/MuseScore/issues/new/choose
Descriptions may need tweaking

### CODE REVIEW

**Comment by bkunda** [comment]
This is great @cbjeukendrup!

I'd just suggest rewording the description under **MuseScore Studio support question** for brevity and clarity:

> If you need help solving a problem in MuseScore Studio, or if you're not certain whether something is a bug, please post your question in the community forum.

**Review by bkunda — CHANGES_REQUESTED** [review]
This is great @cbjeukendrup!

I'd just suggest rewording the description under MuseScore Studio support question for brevity and clarity:

> If you need help solving a problem in MuseScore Studio, or if you're not certain whether something is a bug, please post your question in the community forum.

**Comment by cbjeukendrup** [comment]
@bkunda Done!
